src/presenter/event.js

```diff
@@ -162,7 +168,7 @@ export default class Event {
         UpdateType.MINOR,
         event
     );
-    this._replaceEditToTrip();
+    // this._replaceEditToTrip();
```

Comment:
Давай удалим, это не будет нужно
```suggestion
    // this._replaceEditToTrip();
```